homeassistant/components/zha/core/helpers.py

```diff
@@ -48,7 +48,7 @@
         _LOGGER.debug(
             "%s: bound  '%s' cluster: %s", entity_id, cluster_name, res[0]
         )
-    except DeliveryError as ex:
+    except (DeliveryError, TimeoutError) as ex:
```

Comment:
Is it just the builtin `TimeoutError` we need to catch?

Reply:
I may have been confused by your previous comment. I need to double check now. It’s from future I believe. Was your previous comment just asking why I had _base at the end of the import?

Reply:
Yes!

Reply:
Great catch... I don't think I had enough coffee yesterday or was just having an off day in general. I just confirmed that `concurrent.futures._base.TimeoutError` is the error in the logs and not base timeout. I'll fix this (again) :) thanks again!

Reply:
@MartinHjelmare  it was actually a good thing this happened... found a spot not using the helper that was still broken. I have fixed that as well and tested it. Looks good now.